lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -21,9 +21,49 @@
             <layout.heading local-class='Heading'>
                 <div local-class='Title'>{{t 'registries.drafts.draft.form.new_registration'}}</div>
             </layout.heading>
-            <layout.left local-class='Left'>
-                {{!-- Sidenav: page headings, validations --}}
-            </layout.left>
+            
+            <layout.leftNav as |leftNav|>
+                {{#each draftManager.pageManagers as |pageManager index|}}
+                    <PageLinkManager
+                        @pageManager={{pageManager}}
+                        @pageIndex={{index}}
+                        @currentPage={{draftManager.currentPage}}
+                        as |pageLinkManager|
+                    >
+                        <leftNav.link
```

Comment:
Passing `leftNav.link` to `PageLink` could allow us to move the validation styles from the route to `PageLink`.

Reply:
Or we could create a PageLink component that we pass `PageLinkManager` and `leftNav.link`.